packages/core/src/update/reducer.js

```diff
@@ -15,12 +15,13 @@ import readReducer from '../read/reducer';
 /**
  * Main application reducer.
  *
+ * @param config A configuration object passed from the Engine component
  * @param cursor The cursor representing the state.
  * @param action The action.
  * @returns {*} The new state represented by updated cursor.
  */
-export default function(cursor, action) {
-  // console.log('Reducer: action:', action);
+export default function(config, cursor, action) {
```

Comment:
Not a breaking change since this is not public API. (We need to talk about what is ;)).
First because of currying (the parameters you 'make action upon' always come last, e.g. the list is last for `map`, etc.  . ;)